blint/sbom.py

```diff
@@ -178,18 +179,23 @@ def create_sbom(
     LOG.debug(
         f"SBOM includes {len(components)} components and {len(dependencies)} dependencies",
     )
-    with open(output_file, mode="w", encoding="utf-8") as fp:
-        fp.write(
-            sbom.model_dump_json(
-                indent=None if deep_mode else 2,
-                exclude_none=True,
-                exclude_defaults=True,
-                warnings=False,
-                by_alias=True
-            )
-        )
-        LOG.debug(f"SBOM file generated successfully at {output_file}")
-    return True
+    sbom_content = sbom.model_dump_json(
+                        indent=None if deep_mode else 2,
+                        exclude_none=True,
+                        exclude_defaults=True,
+                        warnings=False,
+                        by_alias=True
+                   )
+    if not output == "json":
+        with open(output, mode="w", encoding="utf-8") as fp:
+            fp.write(sbom_content)
+            LOG.debug(f"SBOM file generated successfully at {output}")
+        return True
+    else:
+        # Write the JSON output directly to stdout
+        sys.stdout.write(sbom_content)
```

Comment:
I think we have to configure the rich logger to use a different file or disable it completely since it might affect the output especially in debug mode.